test/Conversion/CalyxToFSM/lower-invoke.mlir

```diff
@@ -0,0 +1,114 @@
+// RUN: circt-opt --split-input-file -pass-pipeline='builtin.module(calyx.component(lower-calyx-to-fsm))' %s | FileCheck %s
+
```

Comment:
@cgyurgyik  You can see how to invoke component here.I made some changes, I will continue to modify, I have to clarify whether `calyx.invoke` supports the standard library .

Reply:
@rachitnigam @cgyurgyik  I updated this test,in the `lower-invoke` test, I did not use `calyx.group`. I used all `calyx.invoke`.